src/System.Runtime.WindowsRuntime/src/System/InternalHelpers.cs

```diff
@@ -14,3 +17,81 @@ internal static void SetErrorCode(this Exception ex, int code)
         }
     }
 }
+
+namespace Internal.Threading.Tasks
+{
+
+    //
+    // An internal contract that exposes just enough async debugger support needed by the AsTask() extension methods in the WindowsRuntimeSystemExtensions class.
+    //
+    internal static class AsyncCausalitySupport
+    {
+        private static Action<Task> _addToActiveTasks;
+        private static Action<Task> _removeFromActiveTasks;
+        private static Func<bool> _loggingOn;
+        private static Action<Task, string> _traceOperationCreation;
+        private static Action<Task> _traceOperationCompletedSuccess;
+        private static Action<Task> _traceOperationCompletedError;
+
+// Initialize all static fields in 'AsyncCausalitySupport' when those fields are declared and remove the explicit static constructor
+#pragma warning disable CA1810
+        static AsyncCausalitySupport()
+        {
+            Type privateType = typeof(object).Assembly.GetType("Internal.Threading.Tasks.AsyncCausalitySupport");
```

Comment:
Calling this via reflection is not an improvement.

Reply:
I'd also be fine with surfacing this through the `System.Runtime.InteropServices.WindowsRuntime` facade if we felt that was a better choice.